src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -197,5 +201,59 @@ object TestUtils {
 
     ps.executeUpdate()
   }
+
+  def addConsignmentProperty(name: String): Unit = {
+    // name is primary key check exists before attempting insert to table
+    if (!propertyExists(name)) {
```

Comment:
I wonder if it would be better for init.sql to just have a list of standard property names to save before the tests are run?

Reply:
If you want to stick with the current approach you can also add `ON CONFLICT(name) DO NOTHING`, which saves you the extra `SELECT` statement: https://stackoverflow.com/a/31742830/679924

Reply:
Looks like its an H2 Db we are using for the testing. So not sure the `DO NOTHING` statement is available.

Reply:
OK, let's leave it for now.

Like we said on the call yesterday, these tests are getting a bit fragile because of the overlap in test data setup. But this is something that we need to think about more holistically.